### ISSUE TITLE
Repo switcher: Is there a way to disable this? Or move it elsewhere?

### ISSUE THREAD

**Issue by benlk** [ZenHubIO/support#1466]
The "repo switcher" button provided in the ZenHub plugin feels out-of-place. It's not part of ZeHub; it's a separate feature of this plugin that could be pulled out into a separate plugin. It's also mildly duplicative of Github's search's recent-repos list. 

<img width="360" alt="screen shot 2018-07-09 at 6 09 23 pm" src="https://user-images.githubusercontent.com/1754187/42478656-8369ca28-83a3-11e8-9f4e-0fe42ad2408b.png">

Its positioning in the header impinges upon the forked-repo notice:

<img width="310" alt="screen shot 2018-07-09 at 5 56 54 pm" src="https://user-images.githubusercontent.com/1754187/42478288-2c721b22-83a2-11e8-8794-b0c00865cef9.png">

It's kinda vertically aligned with the other buttons in the header, but as a result it seems out-of-alignment with the repo name. Here's a screenshot with element outlines made with Pesticide.io to show that the buttons are sorta aligned:

<img width="914" alt="screen shot 2018-07-09 at 6 02 23 pm" src="https://user-images.githubusercontent.com/1754187/42478376-77edb0de-83a2-11e8-9721-ee9e288e046c.png">

However, on closer inspection, the repo-switcher button is not exactly aligned with the GitHub buttons, and it's a different color. In the following image, I took a screenshot of the repo header and then moved the repo switcher button rightwards until it was close to the GitHub buttons. I didn't move it vertically.

<img width="359" alt="screen shot 2018-07-09 at 6 01 43 pm" src="https://user-images.githubusercontent.com/1754187/42478547-16ea0264-83a3-11e8-95d4-73b0ecb6e49c.png">

Note also that all the buttons in this area have to do with the repository itself. GitHub places the UI for things external to this repository in other places: the black bar header and the very bottom of the page. Maybe you could move the repo-switcher up into the black-bar header, next to notifications or between search and "pull requests"?

<img width="1008" alt="screen shot 2018-07-09 at 6 15 27 pm" src="https://user-images.githubusercontent.com/1754187/42478856-411b3c96-83a4-11e8-8dd9-777e0afc4716.png">

Alternately, could you provide an options page for the browser extension that allows users to disable the repo switcher? The Zenhub plugin could provide an options page through the menu that appears when right-clicking on the plugin's button:

<img width="229" alt="screen shot 2018-07-09 at 6 17 33 pm" src="https://user-images.githubusercontent.com/1754187/42478914-7f705580-83a4-11e8-96d6-40c7bcdbbe11.png">

You'll note that there's a thing there already named "options"; it's greyed-out. Here's some additional information about building options pages for Chrome extensions: https://developer.chrome.com/extensions/getstarted#options

In conclusion: The repo switcher looks and feels out-of-place, and duplicates GitHub features. I'd like for it to disappear.
